src/utils/validateSubmissionFormat.ts

```diff
@@ -0,0 +1,9 @@
+export const validateSubmissionFormat = (message: string) => {
+  const hasBlockOfCode = /```[^`]+```/.test(message);
+  const hasGitHubLink = /github\.com\/[a-zA-Z]/.test(message);
+  const hasLineOfCode = /^`([^`]+)`$/gm.test(message);

```

Comment:
This is not required, we don't need to check that the message has no line of code, we just don't care.